cfgov/tccp/jinja2tags.py

```diff
@@ -0,0 +1,60 @@
+import re
+from urllib.parse import urlparse
+
+from django.template.loader import render_to_string
+from django.utils.safestring import mark_safe
+
+
+# Contact website schemes must be either http or https.
+_valid_website_scheme = re.compile("https?")
+
+
+# Contact website hosts must have at least one . in them.
+_valid_website_netloc = re.compile(r".+\..+")
+
+
+def _format_contact_website(url):
+    parsed_url = urlparse(url)
+
+    if _valid_website_scheme.match(
+        parsed_url.scheme or ""
+    ) and _valid_website_netloc.match(parsed_url.netloc or ""):
+        return {
+            "url": url,
+            "text": f"{parsed_url.scheme}://{parsed_url.netloc}",
```

Comment:
I know this was called out as a change from the previous version, but I'm wondering if we should go back to the link text being just "domain.tld".

At the least I'm thinking we can ditch the scheme from the link text. There are only 4 HTTP URLs in the current dataset, and all of those redirect to HTTPS, so showing the scheme as part of the link text feels like it adds more clutter than usefulness.

I feel less strongly about showing only the hostname and top-level domain and ditching any subdomains. The vast majority of URLs in the current dataset are "www," and it feels like showing a "www" adds no information to the link text. Maybe we can just get rid of "www." in the text string if it's there and leave any other subdomains? Like:

```suggestion
            "text": f"{parsed_url.netloc.replace('www.', '')}",
```

That way we're not assuming a subdomain so URLs like "cadencebank.com" will still display nicely, but we also won't be cluttering the display with information-light things like the "www".

Reply:
@niqjohnson thanks, these are great suggestions, I'll implement.

Reply:
@niqjohnson implemented in c694921.